otel-collector/config.yaml

```diff
@@ -19,16 +31,3 @@ service:
     # Do not remove these extensions
     - health_check
     - zpages
-
-receivers:
-  otlp:
-    protocols:
-      grpc: # port 4317
-      http: # port 4318
-
-extensions:
-  health_check:
-    port: 13133
-  zpages:
-    endpoint: "localhost:55679"
```

Comment:
So we're dropping this on purpose?

Reply:
Nope, I just reognized the file to have service last 😄

Reply:
Oh 😶‍🌫️ 🤦